docs/mindsdb_sql/functions/llm_function.mdx

```diff
@@ -7,12 +7,134 @@ MindsDB provides the `LLM()` function that lets users incorporate the LLM-genera
 
 ## Prerequisites
 
-The `LLM()` function uses one of the OpenAI models and requires the OpenAI API key.
+To use the `LLM()` function with MindsDB, choosing one of the available model providers and define the following environment variables.
 
-To use the `LLM()` function with MindsDB, define the model to be used and the OpenAI API key as environment variables.
+<AccordionGroup>
 
-* The `LLM_FUNCTION_MODEL` environment variable should store the OpenAI model name, like `gpt-4`.
-* The `OPENAI_API_KEY` environment variable should store the OpenAI API key value.
+  <Accordion title="OpenAI">
+    Here are the environment variables for the OpenAI provider:
+
+    ```
+    LLM_FUNCTION_MODEL_NAME
+    LLM_FUNCTION_TEMPERATURE
+    LLM_FUNCTION_MAX_RETRIES
+    LLM_FUNCTION_MAX_TOKENS
+    LLM_FUNCTION_OPENAI_API_BASE
+    LLM_FUNCTION_OPENAI_API_KEY
+    LLM_FUNCTION_OPENAI_ORGANIZATION
+    LLM_FUNCTION_REQUEST_TIMEOUT
+    ```
+
+    Note that the values stored in the environment variables are specific for each provider.
+  </Accordion>
+
+  <Accordion title="Anthropic">
+    Here are the environment variables for the Anthropic provider:
+
+    ```
+    LLM_FUNCTION_MODEL
```

Comment:
I've checked, in get_llm_config all providers expect model_name param (`config.get("model_name",...`)
So variable name here and for other providers should be: LLM_FUNCTION_MODEL_NAME